com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -216,6 +216,10 @@ public void testTf2()
     // once https://github.com/wala/ML/issues/106 is fixed.
     testTf2("tf2_test_model_call3.py", "SequentialModel.call", 1, 4, 2);
     testTf2("tf2_test_model_call4.py", "SequentialModel.__call__", 1, 4, 2);
+    testTf2("tf2_test_callbacks.py", "get_dataset", 0, 0);
+    testTf2("tf2_test_callbacks2.py", "replica_fn", 0, 2);
```

Comment:
There looks to be a tensor parameter for `replica_fn` that is not being picked up by the tensor analysis. Thus, `0` above should be `1`. My guess is that the parameter and argument are not being connected in the XML summary.